datalad/support/annexrepo.py

```diff
@@ -1152,6 +1152,16 @@ def is_crippled_fs(self):
         self.config.reload()
         return self.config.getbool("annex", "crippledfilesystem", False)
 
+    def supports_unlocked_pointers(self):
```

Comment:
should may be it become a `@property`?

Reply:
Fine by me.  I went with a plain method because `is_crippled_fs` right above it was.  Updated in push.